Makefile

```diff
@@ -19,4 +19,4 @@ check:
 	trunk check
 
 test:
-	poetry run pytest -vvv --cov=db_client --cov-fail-under=80
+	poetry run pytest -vvv --cov=db_client --cov-fail-under=80 --cov-report=term --cov-report=html
```

Comment:
Is this going to work in CI?

Reply:
Yeah - I'm playing to try and get this saved as an artifact - so we can see the html output in the browser - but its fiddly

Reply:
Ooooo exciting stuff!!! Woohoo to new CI improvements!!